src/extensions/actions/ActionsRuleCreationModal.tsx

```diff
@@ -286,115 +286,119 @@ export const NeoCustomReportActionsModal = ({
                         <td style={{ paddingLeft: '20px', paddingRight: '20px' }}>
```

Comment:
another one I discovered on extensions.
The rule creation model is slightly off (vertically misaligned elements for a rule).
This happens both for actions and rule-based styling.

Also - For rule-based styling, the rule "Row background color" is never executed.
But probably this was introduced earlier in 2.3.0 -  i dont see it happening in 2.2.5